test/e2e/utils/utils.go

```diff
@@ -73,3 +90,26 @@ func CreateResource(t *testing.T, obj CreateObjects) {
 		t.Fatalf("Failed to create %s: %v", obj.GVR.Resource, err)
 	}
 }
+
+//UpdateResource Create and update the obj and return old and new obj
```

Comment:
```suggestion
// UpdateResource Create and update the obj and return old and new obj
```
Please use a single space at the start of the comment. Please update the commands accordingly.